### ISSUE TITLE
No battery info on Arch

### ISSUE THREAD

**Issue by semiBE** [zakaziko99/agnosterzak-ohmyzsh-theme#8]
Everything went fine so far except the info on battery status.. I am on Arch and nothing displays beside date&time!
Thank you for your work!

Mat

**Comment by nikhilshinday** [comment]
Same, I'm on OSX using iTerm2

**Comment by roberthpereira** [comment]
Same here, i'm on Archlinux using xfce4 terminal

**Comment by afrozalm** [comment]
I had the same problem but installing acpi fixed it.

**Comment by tempestCognitor** [comment]
For arch I installed acpi, but if you're seeing this as well:

prompt_battery:[:31: unknown condition: -gt                                                                                                                                                                        
prompt_battery:[:33: unknown condition: -gt

You need to go to lines 120 and 122 and make the tests double square brackets ([..] -> [[..]])

**Comment by Gy0m** [comment]
i never had battery status on every ubuntu distro i used now i'm on ubuntu 17.10

**Comment by zakaziko99** [comment]
@Gy0m,  the battery status is only shown if the laptop is not charging

**Comment by Gy0m** [comment]
yes sure but i can unplug my battery charger and i never saw that battery status in the prompt. i try to relaunch a new term but it didn't work. I don't know what i miss. It's not a big issue but i still wondering why this don't work

**Comment by Gy0m** [comment]
I resolved the issue by installing `acpi` by `sudo apt install acpi` and now the battery status is displayed. 
Perhap's we could think of adding this step in the README.md ?

> Comment updated 2024 > I wasn't listening to the community upside > Now i'm on ARCH BTW
